Library/Homebrew/livecheck.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Livecheck
+  attr_reader :skip_msg
+
+  def initialize
+    @regex = nil
+    @skip = false
+    @skip_msg = nil
+    @url = nil
+  end
+
+  def regex(pattern = nil)
+    return @regex if pattern.nil?
+
+    @regex = pattern
+  end
+
+  def skip(skip_msg = nil)
+    @skip = true
+    @skip_msg = if skip_msg.blank?
+      nil
+    else
+      skip_msg
+    end
```

Comment:
```suggestion
    @skip_msg = skip_msg.blank? ? nil : skip_msg
```

Since this is so simple (conditionally choosing the value to set), you can use the ternary operator (`?:`) to make this less verbose.